### PR TITLE
Burton 2022 added

### PR DESCRIPTION
**Pull request by TravisGould**

### CODE REVIEW

File: burton_2022/manifest.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+"raw/XXXX_ROIPos.csv" = "X and Y coordinates for the centroids of each ROI (glom) in each experiment XXXX; exported from odormappingdata_fullinfo_06_2021.mat file because Matlab 'table' class is not Python readable."
+"LICENSE" = ""
+
+[processed]
+"main.py" = "Processing workflow."
+"molecules.csv" = "Information about odorant molecules used. -1 CID assigned to 'empty' which is the control."
+"imaging.csv" = "Paths to response maps saved in .csv form for every odorant/mouse combination tested."
+"behavior_1.csv" = "DeltaF response values of every ROI (glomerulus) for every odorant/mouse tested."
+"behavior_2.csv" = "Properties of diagnostic odorants with functionally-identified glomeruli. See main.py for full details."
+"behavior_3.csv" = "Properties of diagnostic odorants that consistently elicit sparse activation but did not meet the cirteria for functional identification. See main.py for full details."
```
Comment (rgerkin) [inline burton_2022/manifest.toml:21]: Typo in criteria.

File: burton_2022/stimuli.csv [hunk 2]
```diff
@@ -0,0 +1,741 @@
+CID,Mouse ID,Conc. (mols/L)
```
Comment (rgerkin) [inline burton_2022/stimuli.csv:1]: Reorganize to have columns:
"Stimulus", "CID", and "Conc. (mols/L)"
Every unique combination of CID and concentration should have a different stimulus ID (could be a simple concatenation with an underscore, up to a certain number of digits).
Add a row for blank.
Drop Mouse ID.

File: burton_2022/molecules.csv [hunk 3]
```diff
@@ -0,0 +1,188 @@
+CID,MolecularWeight,IsomericSMILES,IUPACName,name
+-1,,,,empty
```
Comment (rgerkin) [inline burton_2022/molecules.csv:2]: Drop the -1.

**Review by rgerkin — CHANGES_REQUESTED** [review]
- Also change CID to a stimulus ID everywhere (see stimuli.csv for details).
- Change behavior1 to use stimulus, subject (now experiment, but with same info, i.e. first column of subjects.csv -- a subject is a hemisphere), roi, dF.

**Comment by TravisGould** [comment]
Changes pushed. Except for re-indexing behavior_2 and 3 which aren't set up
to be compatible with a stimulus index. Can be done with some
restructuring, but would be good first to know if there is any data in
those files that you don't want to actually keep.

**Pull request merged**